### PR TITLE
Trigger MC truth fcl

### PR DESCRIPTION
**Pull request by brownd1978**
Add fcl to configure KalSeedMC construction in the trigger reco sequences.  This also fixes a configuration bug which allowed background hits into the pattern recognition.

### CODE REVIEW

**Comment by kutschke** [comment]
@FNALbuild run build test

File: Mu2eKinKal/test/TTTA.fcl
Comment (gianipez) [inline Mu2eKinKal/test/TTTA.fcl:1]: I noticed a tiny bug in this configuration file; when you re-define the trigger sequences, we should add a module instance of the `MergeTriggerInfo` module. This is done automatically at `build` time by the `genTriggerFcl.py` script for generating the Menu files. you would need to add a line like the following to instantiate the module:
`physics.producers."+<trigger_path_name>+"TriggerInfoMerger"+" : { module_type : MergeTriggerInfo }"`

and then of course, append the `<trigger_path_name>+"TriggerInfoMerger"` module instance at the end of the sequence.

File: TrkHitReco/fcl/prolog.fcl
Comment (gianipez) [inline TrkHitReco/fcl/prolog.fcl:17]: I would change the setting of the same parameters in the `makePH` module instance to avoid confusion. 
Also, I think that maybe a de-activation of X straws at smaller(larger) radial distance w.r.t. the solenoid axis would allow to obtain a more uniform selection (I'm not asking to make this change in this PR, but I wanted to mention it to keep it in mind for the future)

File: Mu2eKinKal/test/TTTA.fcl
Comment (brownd1978) [reply Mu2eKinKal/test/TTTA.fcl:1]: This is a diagnostic script used for making TrkAna output.   It needs to be replaced with an official trigger path that includes MC truth matching as we discussed, maybe that's the place to put MergeTriggerInfo.

**Comment by kutschke** [comment]
@brownd1978 are you planning to followup on Giani's comments in this PR or in a follow-on PR?

**Comment by kutschke** [comment]
@FNALbuild run build test

**Comment by kutschke** [comment]
@gianipez please clarify if you are waiting for work from Dave before you approve this.  If not please approve when ready.  Thanks.

**Review by gianipez — APPROVED** [review]

**Pull request merged**
